sycl/plugins/cuda/pi_cuda.hpp

```diff
@@ -376,15 +376,16 @@ struct _pi_queue {
   using native_type = CUstream;
 
   native_type stream_;
+  native_type memStream_;
   _pi_context *context_;
   _pi_device *device_;
   pi_queue_properties properties_;
   std::atomic_uint32_t refCount_;
   std::atomic_uint32_t eventCount_;
 
-  _pi_queue(CUstream stream, _pi_context *context, _pi_device *device,
+  _pi_queue(CUstream stream, CUstream memStream, _pi_context *context, _pi_device *device,
             pi_queue_properties properties)
-      : stream_{stream}, context_{context}, device_{device},
+      : stream_{stream}, memStream_{memStream}, context_{context}, device_{device},
```

Comment:
I would call it "alternativeStream" or something like that , PI Queue class does not need to know what the streams are used for